samples/LowAllocationWebServer/LowAllocationWebServerLibrary/ReadWriteBytes.cs

```diff
@@ -42,37 +42,36 @@ public ReadWriteBytes(Memory<byte> first, IMemorySequence<byte> rest, long lengt
 
         public bool TryGet(ref Position position, out Memory<byte> value, bool advance = true)
         {
-            if (position == Position.First)
+            if (position.IsDefault)
```

Comment:
nit: why not `position == default` like everywhere?

Reply:
Will fix. Thanks,